### DISCUSSION TITLE
after 2+ weeks of farming still no coin

### DISCUSSION THREAD

**Discussion by GeorgePapageorgakis** [Chia-Network/chia-blockchain#1188] [Q&A]
After opening a ticket https://github.com/Chia-Network/chia-blockchain/issues/1184
I have realised that there is insufficient guide for the newcommers here.

Is there something that we are supposed to edit in the config file?
What is the `xch_target_address` in the config file? Is this supposed to be my wallet address? If it is then why does it change every time I reload or restart the app and it is never the same address?

I have almost 5TB with K=32 plots... 2 days estimation yet nothing....

A little guidance would be much appreciated.

**Comment by hoffmang9** [comment]
When you click on the Wiki up there you can find these two items:
https://github.com/Chia-Network/chia-blockchain/wiki/Chia-Keys-Architecture
https://github.com/Chia-Network/chia-blockchain/wiki/FAQ#how-do-i-send-or-receive-a-transaction
